templates/image.qtpl

```diff
@@ -22,7 +24,7 @@
 			<section id="tags">
 				{% if img.Name != "" %}
 					<span class="image-name">
-						<a href="/search?q={%s img.Name %}" title="Search for name">
+						<a href='/search?q={%s url.QueryEscape("name:") + img.Name %}' title="Search for name">
```

Comment:
Why `'`?

Reply:
IDE syntax highlighting gets a bit ugly in .qtpl with nested strings, in my opinion. Changed to be consistent.